modules/nw-ovn-kubernetes-readiness-probes.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-project-readiness-probes_{context}"]
+= OVN-Kubernetes readiness probes
+
+The `ovnkube-master` and `ovnkube-node` pods have containers configured with readiness probes.
```

Comment:
would be nice to link to the section of the docs that describe readinessProbes at https://docs.openshift.com/container-platform/4.12/applications/application-health.html.

Reply:
link added